### DISCUSSION TITLE
Deffered-default.css not including on the page

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#107] [1. Help] [answered]
Hi @vovayatsyuk 

We are facing an issue on our staging site Deffered-default.css not including on the page like home, PLP, PDP, etc but loading on the login page, and this is happening after the upgrade breeze, working fine on Live with a lower version here.

And this is happening with the breeze blank theme as well.

Thanks

**Comment by vovayatsyuk** [comment]
This is your live site output:

```
<link rel="stylesheet" type="text/css" media="print" onload="media='all'" href="...deferred-default.min.css">
```

And this is your staging site:

```
<link rel="preload" media="print" onload="media='all'" href="...deferred-default.min.css" as="style">
```

The version on the live site is generated by Breeze. The version on the staging site is changed by some module I think.

**Comment by vovayatsyuk** [comment, marked as answer]
Maybe it's a swissup/pagespeed critical css? If so, please disable it: http://docs.swissuplabs.com/m2/extensions/pagespeed/configuration/#css-settings-section
